index.html

```diff
@@ -506,8 +506,8 @@
-,{"system":"StarRocks","date":"2023-02-10","machine":"c6a.metal, 500gb gp2","cluster_size":1,"comment":"","tags":["C++","column-oriented","MySQL compatible","ClickHouse derivative"],"load_time":433,"data_size":17705230744,"result":[[0.04,0.04,0.03],[0.12,0.03,0.03],[0.66,0.06,0.06],[2.08,0.08,0.06],[0.10,0.08,0.09],[2.19,0.22,0.21],[0.04,0.02,0.03],[0.06,0.04,0.04],[0.99,0.12,0.12],[0.73,0.74,0.70],[0.13,0.06,0.05],[1.04,0.06,0.07],[0.17,0.18,0.17],[0.23,0.23,0.23],[0.54,0.18,0.18],[0.09,0.09,0.08],[0.31,0.32,0.30],[0.12,0.10,0.11],[1.27,0.48,0.44],[0.01,0.01,0.00],[12.09,0.17,0.15],[0.17,0.13,0.12],[10.91,0.28,0.24],[28.25,0.86,1.36],[0.03,0.03,0.02],[0.06,0.04,0.04],[0.02,0.03,0.02],[0.63,0.24,0.21],[8.77,1.60,1.64],[0.12,0.10,0.10],[1.33,0.13,0.13],[3.46,0.18,0.19],[0.97,0.86,0.86],[0.95,0.96,0.98],[0.96,0.97,0.97],[0.14,0.12,0.14],[0.07,0.06,0.06],[0.05,0.04,0.04],[0.04,0.03,0.03],[0.10,0.09,0.09],[0.47,0.03,0.03],[0.25,0.03,0.03],[0.04,0.03,0.03]],"source":"starrocks/results/c6a.metal.json"}
+,{"system":"StarRocks","date":"2022-09-25","machine":"c6a.4xlarge, 500gb gp2","cluster_size":1,"comment":"","tags":["C++","column-oriented","MySQL compatible","ClickHouse derivative"],"load_time":449,"data_size":16315183154,"result":[[0.043,0.043,0.041],[0.110,0.027,0.027],[0.314,0.054,0.057],[1.594,0.091,0.066],[0.500,0.464,0.466],[1.534,0.843,0.876],[0.088,0.075,0.075],[0.079,0.031,0.031],[1.661,0.869,0.923],[2.006,1.942,1.841],[0.367,0.312,0.316],[0.361,0.311,0.313],[0.584,0.575,0.557],[1.260,1.255,1.191],[0.750,0.789,0.748],[0.820,0.764,0.735],[3.398,3.301,3.281],[2.337,2.247,2.263],[4.089,4.153,4.116],[0.037,0.020,0.021],[12.178,1.310,1.305],[13.071,1.043,1.029],[21.705,1.701,1.684],[11.576,0.856,0.261],[0.113,0.031,0.029],[1.007,0.123,0.126],[0.033,0.030,0.029],[9.758,1.539,1.471],[12.624,13.674,14.134],[0.136,0.060,0.052],[3.237,0.414,0.398],[3.271,0.611,0.601],[4.327,4.305,4.310],[14.588,5.764,5.823],[14.798,5.565,5.535],[0.642,0.543,0.560],[0.184,0.072,0.074],[0.071,0.038,0.040],[0.038,0.034,0.031],[0.191,0.128,0.129],[0.071,0.025,0.025],[0.033,0.025,0.025],[0.036,0.022,0.024]],"source":"starrocks/results/c6a.4xlarge.json"}
```

Comment:
No need to modify index.html, CI updates it automatically based on the results json files.

Reply:
OK, removed.